java.io.FileNotFoundException: https://files.minecraftforge.net/fernflower-fix-1.0.zip

` ./gradlew build` fail

```
> Task :downloadMcpTools FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':downloadMcpTools'.
> java.io.FileNotFoundException: https://files.minecraftforge.net/fernflower-fix-1.0.zip

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.8.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1m 2s
6 actionable tasks: 6 executed
```

--- a/src/main/java/net/minecraftforge/gradle/common/Constants.java
+++ b/src/main/java/net/minecraftforge/gradle/common/Constants.java
@@ -47,7 +47,8 @@ public class Constants {
     public static final String MC_JSON_URL = "https://s3.amazonaws.com/Minecraft.Download/versions/{MC_VERSION}/{MC_VERSION}.json";
     public static final String MC_JAR_URL = "https://s3.amazonaws.com/Minecraft.Download/versions/{MC_VERSION}/{MC_VERSION}.jar";
     public static final String MC_SERVER_URL = "https://s3.amazonaws.com/Minecraft.Download/versions/{MC_VERSION}/minecraft_server.{MC_VERSION}.jar";
-    public static final String MCP_URL = "https://files.minecraftforge.net/fernflower-fix-1.0.zip";
+    @SuppressWarnings("HttpUrlsUsage")
+    public static final String MCP_URL = "http://www.modcoderpack.com/files/mcp908.zip";
     public static final String ASSETS_URL = "https://resources.download.minecraft.net";
     public static final String LIBRARY_URL = "https://libraries.minecraft.net/";
     public static final String FORGE_MAVEN = "https://files.minecraftforge.net/maven";

